docs/src/managing-packages.md

```diff
@@ -208,6 +208,42 @@ For example, the tracked dependencies can be stored inside of the active project
 The whole directory can be moved and `Pkg` will still be able to find the dependencies
 because their path relative to the active project is preserved even though their absolute path has changed.
 
+### Adding a subdir package
+
+If the package you want to add by URL is not in the root of the repository, then you need to manually pass the `subdir` keyword to `Pkg.add`
+or `PackageSpec`. For instance, to add the `SnoopCompileCore` package in the [SnoopCompile](https://github.com/timholy/SnoopCompile.jl)
+repository:
+
+```julia-repl
+julia> Pkg.add(url="https://github.com/timholy/SnoopCompile.jl.git", subdir="SnoopCompileCore")
+     Cloning git-repo `https://github.com/timholy/SnoopCompile.jl.git`
```

Comment:
Does all this output have to be shown? The `Precompiling project...` for example doesn't seem like it adds anything.

Reply:
I only removed the `Precompiling project...` part as I think it's still necessary to show readers that it's installing only `SnoopCompileCore` here.

Reply:
The adding of the `General` registry could also be removed, or?

Reply:
It makes sense. I was aware of this too. I kept it because that's also preserved in some other examples. :joy:

Removed in 610f234

Reply:
Hehe, it probably makes sense to do a full read-through of all examples at some point and make sure not too much info output is shown :)